app/cdash/include/index_functions.php

```diff
@@ -0,0 +1,150 @@
+<?php
```

Comment:
I'm pretty sure this file can be safely deleted. Its content was migrated to `app/cdash/app/Controller/Api/Index.php`.